openapi.yaml

```diff
@@ -16480,14 +16490,18 @@ components:
           type: integer
           minimum: 0
           maximum: 255
-          description: >
-            The priority of the target host for this Record. Lower values are preferred. Only valid and required for SRV record requests.
+          description: |
+            The priority of the target host for this Record. Lower values are preferred. Only valid for
+            MX and SRV record requests. Required for SRV record requests.
+
+            Defaults to `0` if omitted/null for MX record requests.
```

Comment:
I first read "null for MX record requests" as related to null MX records, but it makes sense after rereading.

For clarity, maybe add a sentence saying that the `priority` must be `0` for null MX records?